zephyr/module.yml

```diff
@@ -0,0 +1,4 @@
+name: acpi
```

Comment:
@najumon1980 shouldn't this be `acpica`? I thought we agreed that the ACPI APIs in Zephyr will use the "acpi" namespace, but any references to ACPICA should use "acpica".

Reply:
sure I will update